shells/pipes-shell-2/pipe.js

```diff
@@ -56,7 +56,7 @@ const populateDispatcher = (dispatcher, api, composerFactory, storage, context)
       return await addPipeEntity(msg.entity);
     },
     autofill: async (msg, tid, bus) => {
-      return await autofill(msg, tid, bus, composerFactory, storage, context, portFactory);
+      return await autofill(msg, tid, bus, composerFactory, storage, context, [portFactory.bind(null, bus)]);
```

Comment:
I suggest that pec-port export a `portIndustry`, a function that takes a bus and returns a `portFactory`.

Reply:
done, thanks!